### ISSUE TITLE
Volumes option in cli

### ISSUE THREAD

**Issue by jnummelin** [kontena/kontena#113]
Documentation says that volumes are defined in plural form in kontena.yml: http://www.kontena.io/docs/references/kontena-yml

But in reality CLI uses singular form:
https://github.com/kontena/kontena/blob/master/cli/lib/kontena/cli/stacks/stacks.rb#L140

For sake of consistency, YAML should IMHO use plural form to be consistent with other options. (ports etc.)

**Comment by nevalla** [comment]
I agree. Docker uses plural form, so we should also use it. I will fix this on #112. @jakolehm ok?

**Comment by jakolehm** [comment]
Yes, fix should be targeted for `0.8` that includes refactored cli. Imho this fix only affects `kontena.yml` file because `kontena service create` options should match to `docker run` options (where possible). And `docker run` does not use plural form:

```
$ docker run --help | grep volume
  -v, --volume=[]            Bind mount a volume
  --volumes-from=[]          Mount volumes from the specified container(s)
```

**Comment by nevalla** [comment]
Ok, so let's then use plural form only in `kontena.yml`
